cmd/print.go

```diff
@@ -68,6 +68,10 @@ func printItems(w io.Writer, items []fmt.Stringer, offset int) (err error) {
 }
 
 func printToPager(w io.Writer, buf *bytes.Buffer) (err error) {
+	if noPrompt {
```

Comment:
So, this is somewhat changing the meaning of `noPrompt`. Before, it meant "don't ask for my input", and was only used by `save` when accepting secrets. This is basically overloading it to mean "non-interactive", in order to avoid writing to the pager. That being said, our pager code could probably be improved anyway, for example by inspecting the file descriptor to determine if it's "smart" or not (I'm not sure what the correct technical term is). But, since this isn't that bad of an overloaded meaning, and is blocking another PR, I'm okay with it for now.

Reply:
Hmm good point. I was guiding myself by the `qri --help` where it's description for `--no-prompt` is `--no-prompt          disable all interactive prompts`. I'm fine with it for now though as it's a simple and not too intrusive overload, and we can split it off in the future if the need arises.